Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/utils/WiiUtils.java

```diff
@@ -2,5 +2,13 @@
 
 public final class WiiUtils
 {
+  public static final int RESULT_SUCCESS = 0;
```

Comment:
This seems a bit fragile considering nothing in the C++ source for the enum class suggests the order of the existing values must not be changed and new values must be added at the end. I can easily see somebody (me, for example :) ) forgetting to update the Java code.

What do you think of changing the C++ JNI code to fetch the value of these constants and map the enum class value -> Java enum value?

Reply:
I agree with you, but then I'd need to put some constants in WiiUtils.cpp that are identical to the constants in WiiUtils.java, since I can't access Java constants without jumping through hoops :/

But I suppose it does solve the problem to some degree.

Reply:
Done.